src/languages/swift.js

```diff
@@ -180,6 +180,54 @@ export default function(hljs) {
     ]
   };
 
+  const REGEXP_CONTENTS = [
+    hljs.BACKSLASH_ESCAPE,
+    {
+      begin: /\[/,
+      end: /\]/,
+      relevance: 0,
+      contains: [ hljs.BACKSLASH_ESCAPE ],
+    },
+  ];
+
+  // Adapted from hljs.REGEXP_MODE
+  // https://docs.swift.org/swift-book/documentation/the-swift-programming-language/lexicalstructure/#Regular-Expression-Literals
+  const BARE_REGEXP_LITERAL = {
+    // this outer rule makes sure we actually have a WHOLE regex and not simply
+    // an expression such as:
+    //
+    //     3 / something
+    //
+    // (which will then blow up when regex's `illegal` sees the newline)
+    begin: /(?=\/[^\s][^/\n]*\/)/,
+    contains: [
+      {
+        begin: /\/[^\s]/,
+        end: /\//,
+        illegal: /\n/,
```

Comment:
Is this needed with the "lookahead" you're doing on line 202 which should guarantee there will be no newlines?

Reply:
You're right, this works as intended without this additional rule. This was heavily inspired by `REGEXP_MODE` which uses the [same `illegal` rule](https://github.com/highlightjs/highlight.js/blob/98be55bf6520f62c581a498509baf9387461dce0/src/lib/modes.js#L164). The suite also passes when this `illegal` rule is removed as well, should I include that removal in this patch as well for consistency?

Reply:
Sure, I can't see what purpose it's serving.  I wonder if the whole rule also couldn't be flattened by just lifting the rule out of the contains and adding the lookahead to it's begin?

Reply:
Good suggestion! Yes did the trick much more elegantly